src/main/java/org/snomed/snowstorm/core/data/services/CodeSystemService.java

```diff
@@ -384,7 +384,7 @@ private synchronized void doJoinContentInformation(CodeSystem codeSystem, String
 		contentInformationCache.put(branchPath, Pair.of(latestBranch.getHead(), codeSystem));
 	}
 
-	private void setDependantVersionEffectiveTime(CodeSystem codeSystem, String branchPath, Branch latestBranch) {
+	public void setDependantVersionEffectiveTime(CodeSystem codeSystem, String branchPath, Branch latestBranch) {
```

Comment:
I needed access to this method from `CodeSystemVersionService` so I made it public.